Lesson9/Gokhan/styles.css

```
@@ -0,0 +1,104 @@
+
+/* i added text content, pre element and content centeren on page */
+/* when i was write and save */
+/* i can^t use live server for this work */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100%;
```

Not needed, body element is block elevel element by default
  What you want to do is it remove margin from it
```suggestion
  margin: 0;
```